packages/auth/src/core/strategies/email_and_password.test.ts

```diff
@@ -222,9 +238,73 @@ describe('core/strategies/sendPasswordResetEmail', () => {
       await sendPasswordResetEmail(auth, email);
       expect(apiMock.calls[0].request).to.eql({
         requestType: ActionCodeOperation.PASSWORD_RESET,
-        email
+        email,
+        clientType: 'CLIENT_TYPE_WEB'
       });
     });
+
+    it('calls fallback to recaptcha flow when receiving MISSING_RECAPTCHA_TOKEN error', async () => {
+      if (typeof window === 'undefined') {
+        return;
+      }
+
+      // First call should fail with MISSING_RECAPTCHA_TOKEN error
```

Comment:
nit: "calls without recaptcha token should fail with MISSING_RECAPTCHA_TOKEN error."